redirector.py

```diff
@@ -204,7 +222,7 @@ def main():
 
     settings = dict(
         stats=stats,
-        xsrf_cookies=True,
+        xsrf_cookies=False,
```

Comment:
Yeah, that's why I was like:

![wut](https://cloud.githubusercontent.com/assets/1857993/9891836/4b7770f8-5bbe-11e5-8935-fcf3af506e59.jpg)